hi-ml-cpath/testhisto/testhisto/utils/test_tiles_selection_utils.py

```diff
@@ -42,12 +42,14 @@ def _create_mock_results(n_samples: int, n_tiles: int = 3, n_classes: int = 2, d
     :return: A dictioanry containing randomly generated mock results.
     """
     diff_n_tiles = [n_tiles + i for i in range(n_samples)]
+    probs = torch.rand((n_samples, n_classes), device=device)
+    probs = probs / probs.sum(dim=1, keepdim=True)
```

Comment:
for my own understanding, why this normalization is now needed?

Reply:
also note that we had a bug as True labels were sampled ` torch.randint(2, size=(n_samples,), device=device),` with 2 as upper bound so we never really tested the multi class scenario.